core/src/components/slides/slides.tsx

```diff
@@ -379,7 +403,7 @@ export class Slides {
           <slot></slot>
         </div>
         { this.pager ? <div class="swiper-pagination"></div> : null }
-        { this.scrollbar ? <div class="swiper-scrollbar"></div> : null }
+        <div class="swiper-scrollbar" hidden={!this.scrollbar}/>
```

Comment:
Why is this changed? If the scrollbars are not needed, we should not render them

Reply:
It looks like Swiper expects the scrollbar div to exist in the dom even if it isn't being displayed. When navigating away from a page that has the scrollbar prop set to false you'll receive this error: 
> Uncaught TypeError: Cannot read property 'removeEventListener' of undefined
    at Swiper.disableDraggable (ion-anchor.md.js:8691)
    at Swiper.destroy (ion-anchor.md.js:8770)
    at Swiper.destroy (ion-anchor.md.js:8837)
    at ion-anchor.md.js:3370
    at Array.forEach (<anonymous>)
    at ion-anchor.md.js:3369
    at Array.forEach (<anonymous>)
    at Swiper.emit (ion-anchor.md.js:3363)
    at Swiper.destroy (ion-anchor.md.js:6827)
    at Slides.componentDidUnload (ion-anchor.md.js:11703)

I didn't dig too far into it, there could be a better solution.

Keeping the scrollbar in the dom does seem to silent the `Cannot read property 'removeEventListener' of undefined` errors though.

Reply:
Looks like you need to modify the scrollbar options in the default settings

```
      scrollbar: {
        el: this.scrollbar ? '.swiper-scrollbar' : null,
        hide: true,
      },
```

Then you could go back to conditionally rendering the scrollbar.